admin-dev/themes/new-theme/js/components/grid.js

```diff
@@ -61,7 +60,16 @@ export default class Grid {
       return;
     }
 
-    $gridPanel.find('.test').on('click', () => this._onRefreshClick());
+    let identifier = this.$grid.attr('id');
+    let commonActionSuffix = '#' + identifier + '_action_';
+
+    let refreshListActionId = commonActionSuffix + 'ps_refresh_list';
+    let showSqlActionId = commonActionSuffix + 'ps_show_query';
+    let exportSqlManagerActionId = commonActionSuffix + 'ps_export_sql_manager';
+
+    $gridPanel.on('click', refreshListActionId, () => this._onRefreshClick());
+    $gridPanel.on('click', showSqlActionId, () => this._onShowSqlQueryClick());
+    $gridPanel.on('click', exportSqlManagerActionId, () => this._onExportSqlManagerClick());
```

Comment:
don't we have already a js module that handle SQLManager stuff?

Reply:
we do, with hardcoded catalog page classes, it makes hard to use it, in my opinion that SqlManager should be reworked to make it more flexible, what do you think?

Reply:
I think you're right :+1: